types/const.go

```diff
@@ -20,14 +20,10 @@ const (
 	PriceKVStoreKey        = "price"
 	LinoCoinDenom          = "linocoin"
 
+	// legacy, permissions are no longer used.
 	// Different permission level for msg
-	UnknownPermission                = Permission(0)
-	AppPermission                    = Permission(1)
-	TransactionPermission            = Permission(2)
-	ResetPermission                  = Permission(3)
-	GrantAppPermission               = Permission(4)
-	PreAuthorizationPermission       = Permission(5)
-	AppAndPreAuthorizationPermission = Permission(6)
+	UnknownPermission     = Permission(0)
+	TransactionPermission = Permission(2)
```

Comment:
All permissions can be removed?

Reply:
yes, just the clean-up work is too heavy,  (remove all GetPermission, consumtion in messages).  we can do it later.